packages/core/src/Cardano/types/Cip1854PublicKey.ts

```diff
@@ -0,0 +1,32 @@
+import * as BaseEncoding from '@scure/base';
+import { InvalidStringError, OpaqueString, assertIsBech32WithPrefix, assertIsHexString } from '@cardano-sdk/util';
+
+const MAX_BECH32_LENGTH_LIMIT = 1023;
+export type Cip1854PublicKey = OpaqueString<'Cip1854PublicKey'>;
```

Comment:
I think we need to rename this to:

```suggestion
export type Cip1854ExtendedAccountPublicKey = OpaqueString<'Cip1854ExtendedAccountPublicKey'>;
```
since we are using the prefix `acct_shared_xvk`, Its a bit verbose, but this is different than a PublicKey.

or maybe `Cip1854ExtAcctPublicKey` or `Cip1854Bip32AcctPublicKey`, what do you think @mkazlauskas ?

We should also assert the key is 64 bytes or 128 characters

Reply:
Agree, the regular xpub type is called `Bip32PublicKeyHex`, so this one should be `Cip1854Bip32AcctPublicKeyHex`

Reply:
I'll go with `Cip1854ExtendedAccountPublicKey` since it's more readable than `Cip1854Bip32AcctPublicKeyBech32`

Reply:
Fixed on c66a9c0